### ISSUE TITLE
Enable/Disable "Limit speed & Sync audio" hotkey

### ISSUE THREAD

**Issue by GoogleCodeExporter** [skmp/nulldc#526]
```
Please forgive me if this isn't the right place to suggest this, i've seen some 
similar requests, but they haven't really touched on the purpose.

Basically i'm suggesting the ability to set a hotkey  
(TAB/backspace/insert/end/etc) to enable/disable the already existing option of 
"Limit speed & Sync audio" (without having to access the menu).

That way you can speed up your gameplay on the fly, (very handy for loading 
screens or RPG games).

Basically a "fastforward" option for NullDC

I have no idea how complicated it would be, so thanks in advance for reading 
and all your hard work.
```

Original issue reported on code.google.com by `imK...@gmail.com` on 30 Jan 2015 at 9:56
